### DISCUSSION TITLE
PrepareMagiskFilesTask zipFileName  packaging fails

### DISCUSSION THREAD

**Discussion by Sidy-xiao** [LSPosed/LSPosed#2144] [Q&A] [answered]
Has anyone encountered the prepareMagiskFilesTask task unable to copy packaged scripts? My project KTS packaging will fail, Gradle packaging will work

**Comment by duan998** [comment, marked as answer]
您发给我的邮件已经收到，谢谢！ The email you sent me has been received, thank you!
